packages/vapor/scss/components/facet.scss

```diff
@@ -56,7 +56,9 @@
 }
 
 .facet-header-title {
-    color: var(--facet-header-text-color);
+    --color: var(--title-text-color);
```

Comment:
same here

Reply:
I think here we can use directly var(--title-text-color) since there is no need to override in such a small scope.

Reply:
We should only define `--color` if it's overriden by a mod